### PR TITLE
handle OptionsInvalidNumberProvidedException and return alert inside dashboard

### PR DESCRIPTION
**Pull request by obiefy**

### CODE REVIEW

**Comment by akiyamaSM** [comment]
I guess it would be better if you check if ther are duplicates. If you pass 3 names with 2 duplications it will not throw an exception.

**Comment by obiefy** [comment]
OK, that means we will throw a duplicated exception.

**Comment by akiyamaSM** [comment]
@obiefy Yes, it will throw a  DuplicatedOptionsException

File: src/Http/Controllers/PollManagerController.php [hunk 1]
```diff
@@ -46,10 +46,11 @@ public function store(PollCreationRequest $request)
     {
         try {
             PollHandler::createFromRequest($request->all());
-        } catch (OptionsInvalidNumberProvidedException $th) {
+        } catch (DuplicatedOptionsException $exception) {
+            dd($request->all());
```
Comment (akiyamaSM) [inline src/Http/Controllers/PollManagerController.php]: remove the dd

**Comment by obiefy** [comment]
Done.

**Comment by akiyamaSM** [comment]
Thanks @obiefy for the PR.

**Pull request merged**